@utahdts/utah-design-system/react/components/widgetsIndicators/Skeleton.jsx

```diff
@@ -0,0 +1,20 @@
+import { joinClassNames } from '../../util/joinClassNames';
+import { skeletonStyles } from '../../enums/skeletonStyles';
+
+/**
+ * @param {object} props
+ * @param {string} [props.className]
+ * @param {import('react').RefObject<HTMLDivElement>} [props.innerRef]
+ * @param {string} [props.style]
+ * @returns {import('react').JSX.Element}
+ */
+export function Skeleton({
+  className,
+  innerRef,
+  style = skeletonStyles.RECTANGULAR,
```

Comment:
`style` is an already existing prop on html components.
```<div style={{color:'red'}}>I am green!</div>```
Repurposing the `style` property may be confusing?